### PR TITLE
zh: Update GKE terraform deployment docs

### PR DESCRIPTION
**Pull request by cofyc**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v1.1 changes for now)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/deploy-on-gcp-gke.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ category: how-to
 * [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
 * [Google Cloud SDK](https://cloud.google.com/sdk/install)
 * [Terraform](https://www.terraform.io/downloads.html) >= 0.12
-* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl) >= 1.14
-* [Helm](https://helm.sh/docs/using_helm/#installing-the-helm-client) >= 2.9.0 且 < 3.0.0
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl) >= 1.12
+* [Helm](https://helm.sh/docs/using_helm/#installing-the-helm-client) >= 2.11.0 且 < 3.0.0
```
Comment (cofyc) [inline zh/deploy-on-gcp-gke.md]: same as aws docs.

File: zh/deploy-on-gcp-gke.md [hunk 2]
```diff
@@ -235,51 +278,6 @@ Terraform 自动加载和填充匹配 `terraform.tfvars` 或 `*.auto.tfvars` 文
     helm ls
     ```
 
-## 升级 TiDB 集群
```
Comment (cofyc) [inline zh/deploy-on-gcp-gke.md]: this is the same in all k8s environments, we don't need to doc it here.

File: zh/deploy-on-gcp-gke.md [hunk 3]
```diff
@@ -369,10 +349,7 @@ kubectl --kubeconfig credentials/kubeconfig_<gke_cluster_name> get po -n <tidb_c
 例如，可以将 `tidb_count` 从 1 改为 2 来扩容 TiDB：
 
 ```hcl
-variable "tidb_count" {
-  description = "Number of TiDB nodes per availability zone"
-  default     = 2
-}
+tidb_count = 2
```
Comment (cofyc) [inline zh/deploy-on-gcp-gke.md]: recommend users to configure via `.tfvars` file

File: zh/deploy-on-gcp-gke.md [hunk 4]
```diff
@@ -132,6 +132,8 @@ Terraform 自动加载和填充匹配 `terraform.tfvars` 或 `*.auto.tfvars` 文
 
         如上所述，生产环境的部署需要 91 个 CPU，超过了 GCP 项目的默认配额。可以参考[配额](https://cloud.google.com/compute/quotas)来增加项目配额。扩容同样需要更多 CPU。
 
+		- 默认创建的是 Regional 集群，会在 3 个可用区里都创建节点数量。比如配置 `pd_count = 1` ，实际为 PD 创建的节点数为 3 个。可以通过配置 `location` 来创建 Zonal 集群，具体可参见 `examples/` 下例子。
```
Comment (ran-huang) [inline zh/deploy-on-gcp-gke.md]:
```suggestion
		- 默认创建的是 Regional 集群，会在 3 个可用区里都创建节点数量。比如配置 `pd_count = 1`，实际为 PD 创建的节点数为 3 个。可以通过配置 `location` 来创建 Zonal 集群，具体可参见 `examples/` 下例子。
```

File: zh/deploy-on-gcp-gke.md [hunk 5]
```diff
@@ -155,29 +157,69 @@ Terraform 自动加载和填充匹配 `terraform.tfvars` 或 `*.auto.tfvars` 文
...
+
+    ```shell
+	  cp manifests/{db,db-monitor}.yaml .
+    ```
+
+    参考 [API 文档](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.html)和[集群配置文档](configure-cluster-using-tidbcluster.md)完成 CR 文件配置。
+
+    > **注意：**
+    >
+    > * 请使用 GKE 部署过程中配置的 `default_tidb_cluster_name` (默认为 `tidb-cluster`) 替换 `db.yaml` 和 `db-monitor.yaml` 文件中所有的 `CLUSTER_NAME`。
```
Comment (ran-huang) [inline zh/deploy-on-gcp-gke.md]:
```suggestion
    > * 请使用 GKE 部署过程中配置的 `default_tidb_cluster_name`（默认为 `tidb-cluster`）替换 `db.yaml` 和 `db-monitor.yaml` 文件中所有的 `CLUSTER_NAME`。
```

File: zh/deploy-on-gcp-gke.md [hunk 6]
```diff
@@ -155,29 +157,69 @@ Terraform 自动加载和填充匹配 `terraform.tfvars` 或 `*.auto.tfvars` 文
...
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl --kubeconfig credentials/kubeconfig_<gke_name> create namespace <namespace>
+    ```
+
+    > **注意：**
+    >
+    > `namespace` 是[命名空间](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)，可以起一个方便记忆的名字，比如和 `default_cluster_name` 相同的名称。
+3. 部署 TiDB 集群：
```
Comment (ran-huang) [inline zh/deploy-on-gcp-gke.md]:
```suggestion

3. 部署 TiDB 集群：
```

File: zh/deploy-on-gcp-gke.md [hunk 7]
```diff
@@ -155,29 +157,69 @@ Terraform 自动加载和填充匹配 `terraform.tfvars` 或 `*.auto.tfvars` 文
...
+  kubectl --kubeconfig credentials/kubeconfig_<gke_name> create -f db.yaml -n <namespace>
+  kubectl --kubeconfig credentials/kubeconfig_<gke_name> create -f db-monitor.yaml -n <namespace>
+  ```
+
 ## 访问 TiDB 数据库
 
 `terraform apply` 运行完成后，可执行以下步骤来访问 TiDB 数据库。注意用[部署 TiDB 集群](#部署-tidb-集群)小节的输出信息替换 `<>` 部分的内容。
 
-1. 通过 `ssh` 远程连接到堡垒机。
+1. 获取 tidb Internal LoadBalancer IP 地址:
```
Comment (ran-huang) [inline zh/deploy-on-gcp-gke.md]:
```suggestion
1. 获取 TiDB Internal LoadBalancer IP 地址:
```

File: zh/deploy-on-gcp-gke.md [hunk 8]
```diff
@@ -155,29 +157,69 @@ Terraform 自动加载和填充匹配 `terraform.tfvars` 或 `*.auto.tfvars` 文
...
+
+	```shell
```
Comment (ran-huang) [inline zh/deploy-on-gcp-gke.md]: Don't use hard tab to indent. Use 4 spaces. Please also replace other tabs in this document with spaces. Thanks!
```suggestion
    ```shell
```

File: zh/deploy-on-gcp-gke.md [hunk 9]
```diff
@@ -187,7 +229,8 @@ Terraform 自动加载和填充匹配 `terraform.tfvars` 或 `*.auto.tfvars` 文
 
     > **注意：**
     >
-    > 通过 MySQL 连接 TiDB 前，需要先安装 MySQL 客户端。
+    > 通过 MySQL 连接 TiDB 前，需要先安装 MySQL 客户端。CentOS 系统可通过 `sudo yum install -y mysql` 安装。
+    > `<tidb_ilb_ip` 为前面获取的 Internal LoadBalancer IP 地址。
```
Comment (ran-huang) [inline zh/deploy-on-gcp-gke.md]:
```suggestion
    > `<tidb_ilb_ip>` 为前面获取的 Internal LoadBalancer IP 地址。
```

File: zh/deploy-on-gcp-gke.md [hunk 10]
```diff
@@ -292,57 +290,39 @@ Check Table Before Drop: false
...
-    {{< copyable "" >}}
-
-    ```hcl
-    output "how_to_connect_to_example_tidb_cluster_from_bastion" {
-    value = module.example-tidb-cluster.how_to_connect_to_tidb_from_bastion
-    }
-    ```
-
-    上述配置可使该脚本打印出用于连接 TiDB 集群的命令。
+    > - 为任一组件实际创建的总节点数等于配置文件中的节点数乘以配置的可用区的个数 (Regional 集群默认为 3)。
```
Comment (ran-huang) [inline zh/deploy-on-gcp-gke.md]:
```suggestion
    > - 为任一组件实际创建的总节点数等于配置文件中的节点数乘以配置的可用区的个数（Regional 集群默认为 3）。
```

File: zh/deploy-on-gcp-gke.md (hunk 4 above)
Comment (DanielZhangQD) [reply zh/deploy-on-gcp-gke.md]: Can we combine this with the note in L139?

File: zh/deploy-on-gcp-gke.md [hunk 11]
```diff
@@ -155,29 +157,69 @@ Terraform 自动加载和填充匹配 `terraform.tfvars` 或 `*.auto.tfvars` 文
...
     ```
 
+## 部署 TiDB 集群和监控
+
+1. 准备 TidbCluster 和 TidbMonitor CR 文件：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+	  cp manifests/{db,db-monitor}.yaml .
```
Comment (DanielZhangQD) [inline zh/deploy-on-gcp-gke.md]: The files have suffix `example` in the repo.

File: zh/deploy-on-gcp-gke.md [hunk 12]
```diff
@@ -155,29 +157,69 @@ Terraform 自动加载和填充匹配 `terraform.tfvars` 或 `*.auto.tfvars` 文
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl --kubeconfig credentials/kubeconfig_<gke_name> create namespace <namespace>
+    ```
+
+    > **注意：**
+    >
+    > `namespace` 是[命名空间](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)，可以起一个方便记忆的名字，比如和 `default_cluster_name` 相同的名称。
```
Comment (DanielZhangQD) [inline zh/deploy-on-gcp-gke.md]: `default_tidb_cluster_name`

File: zh/deploy-on-gcp-gke.md [hunk 13]
```diff
@@ -155,29 +157,69 @@ Terraform 自动加载和填充匹配 `terraform.tfvars` 或 `*.auto.tfvars` 文
...
+
 ## 访问 TiDB 数据库
 
 `terraform apply` 运行完成后，可执行以下步骤来访问 TiDB 数据库。注意用[部署 TiDB 集群](#部署-tidb-集群)小节的输出信息替换 `<>` 部分的内容。
 
-1. 通过 `ssh` 远程连接到堡垒机。
+1. 获取 tidb Internal LoadBalancer IP 地址:
+
+	```shell
+  kubectl --kubeconfig credentials/kubeconfig_<gke_name> get svc <cluster-name>-tidb
```
Comment (DanielZhangQD) [inline zh/deploy-on-gcp-gke.md]: -n \<namespace\>

File: zh/deploy-on-gcp-gke.md [hunk 14]
```diff
@@ -292,57 +290,39 @@ Check Table Before Drop: false
...
-    tidb_cluster_chart_version = "v1.0.0"
-    pd_instance_type           = "n1-standard-1"
-    tikv_instance_type         = "n1-standard-4"
-    tidb_instance_type         = "n1-standard-2"
-    monitor_instance_type      = "n1-standard-1"
-    pd_node_count              = 1
-    tikv_node_count            = 2
-    tidb_node_count            = 1
-    monitor_node_count         = 1
+			providers = {
```
Comment (DanielZhangQD) [inline zh/deploy-on-gcp-gke.md]: Too much indent here.

File: zh/deploy-on-gcp-gke.md (hunk 3 above)
Comment (DanielZhangQD) [reply zh/deploy-on-gcp-gke.md]: Yes, can we make it clear above this block that users can set this in .tfvars?

File: zh/deploy-on-gcp-gke.md [hunk 15]
```diff
@@ -580,7 +557,6 @@ terraform destroy
 在 TiDB 的案例中，Terraform 模块通常结合了几个子模块：
 
 - `tidb-operator`：为 TiDB 集群提供 [Kubernetes Control Plane](https://kubernetes.io/docs/concepts/#kubernetes-control-plane) 并部署 TiDB Operator。
-- `tidb-cluster`：在目标 Kubernetes 集群中创建资源池并部署 TiDB 集群。
```
Comment (DanielZhangQD) [inline zh/deploy-on-gcp-gke.md]: Keep it and change to `- `tidb-cluster`：在目标 Kubernetes 集群中创建资源池。`, is it OK?

Comment (cofyc) [reply zh/deploy-on-gcp-gke.md]: sure, updated.

File: zh/deploy-on-gcp-gke.md (hunk 3 above)
Comment (cofyc) [reply zh/deploy-on-gcp-gke.md]: sure, updated

File: zh/deploy-on-gcp-gke.md (hunk 14 above)
Comment (cofyc) [reply zh/deploy-on-gcp-gke.md]: fixed

File: zh/deploy-on-gcp-gke.md (hunk 13 above)
Comment (cofyc) [reply zh/deploy-on-gcp-gke.md]: fixed

File: zh/deploy-on-gcp-gke.md (hunk 12 above)
Comment (cofyc) [reply zh/deploy-on-gcp-gke.md]: fixed

File: zh/deploy-on-gcp-gke.md (hunk 11 above)
Comment (cofyc) [reply zh/deploy-on-gcp-gke.md]: fixed

File: zh/deploy-on-gcp-gke.md [hunk 16]
```diff
@@ -292,57 +291,39 @@ Check Table Before Drop: false
...
-    {{< copyable "" >}}
-
-    ```hcl
-    output "how_to_connect_to_example_tidb_cluster_from_bastion" {
-    value = module.example-tidb-cluster.how_to_connect_to_tidb_from_bastion
-    }
-    ```
-
-    上述配置可使该脚本打印出用于连接 TiDB 集群的命令。
+    > - 为任一组件实际创建的总节点数等于配置文件中的节点数乘以配置的可用区的个数（Regional 集群默认为 3）。
```
Comment (ran-huang) [inline zh/deploy-on-gcp-gke.md]: How about this?
```suggestion
    > - 为任一组件实际创建的总节点数 = 配置文件中的节点数 * 配置的可用区的个数（Regional 集群默认为 3）。
```

File: zh/deploy-on-gcp-gke.md [hunk 17]
```diff
@@ -132,6 +132,8 @@ Terraform 自动加载和填充匹配 `terraform.tfvars` 或 `*.auto.tfvars` 文
 
         如上所述，生产环境的部署需要 91 个 CPU，超过了 GCP 项目的默认配额。可以参考[配额](https://cloud.google.com/compute/quotas)来增加项目配额。扩容同样需要更多 CPU。
 
+    - 默认创建的是 Regional 集群，会在 3 个可用区里都创建节点数量。比如配置 `pd_count = 1`，实际为 PD 创建的节点数为 3 个。可以通过配置 `location` 来创建 Zonal 集群，具体可参见 `examples/` 下例子。
```
Comment (DanielZhangQD) [inline zh/deploy-on-gcp-gke.md]: Can we combine this with the note in L139?

File: zh/deploy-on-gcp-gke.md [hunk 18]
```diff
@@ -155,29 +157,70 @@ Terraform 自动加载和填充匹配 `terraform.tfvars` 或 `*.auto.tfvars` 文
...
     ```
 
+## 部署 TiDB 集群和监控
+
+1. 准备 TidbCluster 和 TidbMonitor CR 文件：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    cp manifests/{db,db-monitor}.yaml.example .
```
Comment (DanielZhangQD) [inline zh/deploy-on-gcp-gke.md]: May also rename the files here, the following procedures refer to the files without `example`.

File: zh/deploy-on-gcp-gke.md [hunk 19]
```diff
@@ -292,57 +291,39 @@ Check Table Before Drop: false
...
-    ```
-
-    上述配置可使该脚本打印出用于连接 TiDB 集群的命令。
+    > - 为任一组件实际创建的总节点数等于配置文件中的节点数乘以配置的可用区的个数（Regional 集群默认为 3）。
 
 2. 修改完成后，执行以下命令来创建集群。
 
     {{< copyable "shell-regular" >}}
 
-    ```bash
```
Comment (DanielZhangQD) [inline zh/deploy-on-gcp-gke.md]: This is required since we add a new module in the above step.

File: zh/deploy-on-gcp-gke.md (hunk 16 above)
Comment (cofyc) [reply zh/deploy-on-gcp-gke.md]: sure, updated

File: zh/deploy-on-gcp-gke.md (hunk 18 above)
Comment (cofyc) [reply zh/deploy-on-gcp-gke.md]: done, add some instructions to replace `CLUSTER_NAME` and rename the files

File: zh/deploy-on-gcp-gke.md (hunk 19 above)
Comment (cofyc) [reply zh/deploy-on-gcp-gke.md]: done

File: zh/deploy-on-gcp-gke.md (hunk 17 above)
Comment (cofyc) [reply zh/deploy-on-gcp-gke.md]: moved it to `注意`

**Pull request merged**